SoftLayer/CLI/block/subnets/assign.py

```diff
@@ -0,0 +1,31 @@
+"""Assign block storage subnets to the given host id."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('access_id')
+@click.option('--subnet-id', multiple=True, type=int,
+              help="ID of the subnets to assign; e.g.: --subnet-id 1234")
+@environment.pass_env
+def cli(env, access_id, subnet_id):
+    """Assign block storage subnets to the given host id.
+
+    access_id is the allowed_host_id from slcli block access-list
+    """
+    subnets_id = list(subnet_id)
+    block_manager = SoftLayer.BlockStorageManager(env.client)
+    assigned_subnets = block_manager.assign_subnets_to_acl(access_id,
+                                                           subnets_id)
+
+    for subnet in assigned_subnets:
+        click.echo("Successfully assigned subnet id: " + str(subnet) +
```

Comment:
I'd prefer the string.format() style, reads a bit easier.

```python
click.echo("Successfully assigned subnet id: {} to allowed host id: {}.".format(subnet, access_id))
```

You can also use 120 character long lines.

Reply:
will update.